### ISSUE TITLE
Can you please provide instructions to setup up the image stream input 

### ISSUE THREAD

**Issue by Likhith-Sugganahalli** [surfii3z/packnet_sfm_ros#4]
Can you please provide the instructions to setup image stream for the inference input

**Comment by surfii3z** [comment]
I use ROS for running the system. So I simply publish image messages to

https://github.com/surfii3z/packnet_sfm_ros/blob/8c68799f6f39dcc6cbd8eda9e89cf7f2aeee6975/ros/trt_packnet_node#L31

Otherwise, you should modify this source file https://github.com/surfii3z/packnet_sfm_ros/blob/8c68799f6f39dcc6cbd8eda9e89cf7f2aeee6975/scripts/infer.py

to accept the image stream using OpenCV, for example.

**Comment by Likhith-Sugganahalli** [comment]
thank you for your help, i have set up packnet_sfm_node and receive a depth map, but am confused on the difference between the packnet_sfm_node and trt_packnet_sfm, is the difference only in different implementations? or is there a performance difference.

sorry if this is a basic question, this is new to me and am learning as a go.

**Comment by surfii3z** [comment]
Hi @Likhith-Sugganahalli.

tl; dr It is the same, but trt_packnet_sfm is faster.

In trt_packnet_sfm, you need to use the model in TensorRT format `.trt`. It is just some accelerated implementation of packnet_sfm_node. In my case, I was able to run at `18 FPS at 384x288 on Tesla V100` comparing to `18 FPS at 384x288 on Tesla V100` without TensorRT acceleration.

I also provide the script, please refer to Readme.md

**Comment by Likhith-Sugganahalli** [comment]
thank you
